core/src/main/scala/sttp/tapir/internal/TapirFile.scala

```diff
@@ -0,0 +1,7 @@
+package sttp.tapir.internal
+
+import sttp.tapir.{RangeValue, TapirFileCompanionExtensions, TapirFileExtension}
+
+case class TapirFile(underlying: Any, range: Option[RangeValue] = None) extends TapirFileExtension
```

Comment:
I think I'd use a different range class here, as `RangeValue` is connected with a header value - it has additional info which is useless here. We just need the start/end bytes